src/components/Books.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import AddBooks from './AddBooks';
+import Book from './Book';
+
+const Books = () => (
+  <main className="d-flex flex-column container mt-5">
+    <section className="d-flex flex-row col-12 col-md-8 justify-content-between align-items-center mx-auto mb-5">
+      <Book title="Book title" author="Book author" />
+      <button type="button">Remove</button>
+    </section>
+    <AddBooks />
+  </main>
```

Comment:
- [ ] kindly consider rendering this <Book /> component in a map function. this would make your code ready for next milestones. 
here is the code that you can use and adjust in this component.
```
const books=[
  {title: "ABC", author:"xyz"}
  {title: "ABC1", author:"xyz1"}
  {title: "ABC2", author:"xyz2"}
]
  return (
{books.map((book,index)=>(
 <div key={index}>
     <Book title={book.title} author={book.author}/>
 </div>
))}
  );
```